CHANGELOG.md

```diff
@@ -20,22 +20,24 @@ Changes are grouped as follows
 - Separate read/write fields on data classes
 
 ## [Unreleased]
+
+## [1.0.5] - 2019-08-15
 ### Changed
-- Fixed naming for sequencesdata iteritems() -> items()
-- Will chunk raw rows into batches of 10,000 instead of 1,000.
+- Renamed sequences.data.iteritems() to .items()
+- raw.insert() now chunks raw rows into batches of 10,000 instead of 1,000
 
 ### Fixed
-- Sequence POST queries are retried if safe (e.g. search)
-- .update in the different APIs now correctly accepts a CogniteResourceList
+- Sequences queries are now retried if safe
+- .update() in the different APIs now correctly accept a CogniteResourceList as input
 - Sequences datapoint retrieval updated to use the new cursor feature in the API
 - Json serializiation in `__str__()` of base data classes. Now handles Decimal and Number objects. 
 
 ### Added
-- The files API can now create a file without uploading content.
-- Added parameter recursive to raw.databases.delete() for recursively deleting tables.
+- files.create() method to enable creating a file without uploading content.
+- `recursive` parameter to raw.databases.delete() for recursively deleting tables.
 
 ### Fixed
```

Comment:
Just noticed that we have two ### Fixed headers. Merge the two?

Reply:
will do 👍